starters/blog/src/components/bio.js

```diff
@@ -23,7 +23,10 @@ const Bio = () => {
       }
       site {
         siteMetadata {
-          author
+          author {
+            name
+            summary
+          }
```

Comment:
There is one more usage of `author` in this file:

https://github.com/gatsbyjs/gatsby/blob/b25b52b51746831d051443fc7c66f682eacf7f6e/starters/blog/src/components/bio.js#L46-L48

Should be also changed to `alt={author.name}`